### ISSUE TITLE
release candidate handin client not working with assignments that have checkers

### ISSUE THREAD

**Issue by jbclements** [racket/handin#37]
It looks like the handin-server isn’t working right with newer versions of DrRacket. 

To reproduce: 

1) Start an up-to-date handin server, accepting an assignment (I’ll call it zzz) which contains a checker.rkt that looks like this:

``` racket
(module checker handin-server/checker

(check: :language '(module racket)

      ))
```

2) open an earlier version of DrRacket, e.g. 7.4.

3) Enter this program

#lang racket

1234

4) Submit to the handin server, see everything work smoothly.

5) Quit, open DrRacket HEAD or v7.4.0.902

6) Submit the same file to the handin server, see the following error in the log and in DrRacket:

[8|2019-11-05T12:39:53] ERROR: vector-set!: index is out of range for empty vector
[8|2019-11-05T12:39:53]   index: 0

I first observed this with an older version of the handin server (racket v7.4 + handin server commit 233b17e820bdd9b859f5). I then tried updating the handin server to commit cbb4523b88adb0415c2f, and this had no effect on the problem. I then tried rolling the handin-client installation *back* to 233b17e820bdd9b859f5, but this also did not solve the problem.

Short version: as far as I can tell, the release candidate does not support the use of the handin server except with assignments that have no checker whatsoever.

(FWIW, I searched the handin-server for uses of `vector-set!`, and didn't see any. Too bad.)

**Comment by mflatt** [comment]
Short answer: DrRacket v7.5 changed the WXME format, and Racket v7.4 and doesn't handle the new format. 

Unfortunately, Racket v7.4 also doesn't check the WXME version as intended, so the error message is bad. The version-checking bug has been fixed, so the error message should be right if the format ever changes again.

But the fact that a v7.5 client can't submit to a v7.4 server is not so much a bug as a drawback of changing the WXME format to improve image handling.

**Comment by jbclements** [comment]
Okay, sounds good. I'll update the version of racket running on the server side, and I'll close this bug.
